webui/src/pages/list/ListQuery.tsx

```diff
@@ -95,6 +98,13 @@ const useStyles = makeStyles<Theme, StylesProps>((theme) => ({
       padding: theme.spacing(2, 3),
     },
   },
+  gitbugButton: {
```

Comment:
it's a little weird to have the `gitbug` namespace here. Is there a reason? If you can't use `button`, maybe `newButton` or something?

Reply:
Well... I agree with You. If I rename to "greenButton" will It be ok for You?
I would like to use this to apply the green style (normal and hover) in all situations it is indicated, as You can see in the screenshots.
May I do it when fixing #539 which is another simple issue?

Reply:
I guess `greenButton` works.
If you do #539 in the same branch/PR it's fine, otherwise it's best to fix the code here so it can be merged.

Reply:
I did a commit right now. ;-)